Selenium/ES-12/test_es12ratepoistatus.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Selenium IDE
+import pytest
+import time
+import json
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+class TestEs12ratepoistatus2():
+  def setup_method(self, method):
+    self.driver = webdriver.Chrome()
+    self.vars = {}
+  
+  def teardown_method(self, method):
+    self.driver.quit()
+  
+  def test_es12ratepoistatus2(self):
+    self.driver.get("http://localhost:3000/map")
+    self.driver.set_window_size(1936, 576)
+    time.sleep(2)
+    element = self.driver.find_element(By.CSS_SELECTOR, ".toilets-marker-icon")
+    actions = ActionChains(self.driver)
+    actions.move_to_element(element).click_and_hold().perform()
+    element = self.driver.find_element(By.CSS_SELECTOR, ".toilets-marker-icon")
+    actions = ActionChains(self.driver)
+    actions.move_to_element(element).perform()
+    element = self.driver.find_element(By.CSS_SELECTOR, ".toilets-marker-icon")
+    actions = ActionChains(self.driver)
+    actions.move_to_element(element).release().perform()
+    self.driver.find_element(By.CSS_SELECTOR, ".toilets-marker-icon").click()
+    self.driver.find_element(By.CSS_SELECTOR, ".toilets-marker-icon").click()
+    self.driver.find_element(By.CSS_SELECTOR, ".show-details").click()
+    self.driver.find_element(By.CSS_SELECTOR, ".btn-success").click()
+    assert self.driver.switch_to.alert.text == "You have already rated this POI, please wait -24h-47m-20s to rate its status again!"
+    self.driver.find_element(By.CSS_SELECTOR, ".btn-error").click()
+    assert self.driver.switch_to.alert.text == "You have already rated this POI, please wait -24h-47m-25s to rate its status again!"
```

Comment:
These asserts will probably fail most of the time. Remove the timestamp from the test (maybe just check for the beggining of the sentence)